x/margin/types/params.go

```diff
@@ -36,6 +65,30 @@ func DefaultParams() Params {
 	return NewParams()
 }
 
+// ParamSetPairs get the params.ParamSet
+func (p *Params) ParamSetPairs() paramtypes.ParamSetPairs {
```

Comment:
@cosmic-vagabond 
Something wired. I got removed all these param set in the previous PR. How could this appear again?

Reply:
@kenta-elys I added them back as they are required to initiate the key table in the upgrade handle function

Reply:
Got it.